### ISSUE TITLE
[source.0] Additional properties are not allowed ('priority' was unexpected)

### ISSUE THREAD

**Issue by kianmeng** [dunossauro/videomaker-helper#4]
```sh
$ poetry install

The Poetry configuration is invalid:
  - [source.0] Additional properties are not allowed ('priority' was unexpected)

$ poetry --version
Poetry (version 1.3.2)
```

**Comment by dunossauro** [comment]
Hello @kianmeng, the source priority was released on poetry 1.5.0

https://python-poetry.org/docs/repositories/#supplemental-package-sources

E need created documentation for contributors, sorry for that

**Comment by kianmeng** [comment]
Thanks I will close this issue.
